demand-capacity-mgmt-backend/src/main/java/org/eclipse/tractusx/demandcapacitymgmt/demandcapacitymgmtbackend/appinfo/InfoConfiguration.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2024-25 Smart Sense Consulting Solutions Pvt. Ltd.
```

Comment:
This is not a valid Copyright and License header for Eclipse-Tractusx, please refer to [TRG 7.02 - License and Copyright header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02)

Reply:
fixed with https://github.com/eclipse-tractusx/demand-capacity-mgmt/pull/72/commits/5dae4a27379df156672d326210033489a710b240

Reply:
@carslen Can you please mark this as a resolve if done?